src/InfoBlock.jsx

```diff
@@ -0,0 +1,12 @@
+import style from './InfoBlock.module.css';
+
+function InfoBlock(props) {
+  return (
+    <>
+      <h2>{props.h}</h2>
+      <p className={style.p}>{props.p}</p>
```

Comment:
Please avoid using one letter variable names. There are programming languages where it might be a recommended practice, but not in JavaScript.